content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -168,16 +177,34 @@ topologyKey: kubernetes.io/hostname
 whenUnsatisfiable: ScheduleAnyway
 ```
 
+{{% notice note %}}
+topologyKey refers to the key of KubeVirt infra node labels.
+{{% /notice %}}
+
 this allows us to spread Virtual Machine equally across a cluster.
 
+**Advanced settings: customize Topology Spread Constraints and Node Affinity**
+
 However, it is possible to change the default behaviour and create your own topology combined with Node Affinity Presets.
 You can do it by expanding *ADVANCED SCHEDULING SETTINGS* on the initial nodes dashboard page.
 
 ![Instance Types and Preferences](/img/kubermatic/main/architecture/supported-providers/kubevirt/scheduling-form.png?classes=shadow,border "Advanced Scheduling Settings")
 
+{{% notice note %}}
+Node Affinity Preset Key: refers to the key of KubeVirt infra node labels.
+
+Node Affinity Preset Values: refers to the values of KubeVirt infra node labels.
+{{% /notice %}}
+
+Node Affinity Preset type is: "hard" or "soft" and refers to the same notion of [Pod affinity/anti-affinity types](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#types-of-inter-pod-affinity-and-anti-affinity):
+- "hard": the scheduler can't schedule the VM  unless the rule is met.
+- "soft": the scheduler tries to find a node that meets the rule. If a matching node is not available, the scheduler still schedules the VM. 
+
 It gives you a possibility to create your own unique scheduling options that override ours.
 For instance, you could avoid creation of Virtual Machines on database nodes etc.
 
+Note that you can specify a `Node Affinity Preset Key` and leave `Node Affinity Preset Values` empty to constrain the VM to run on KubeVirt infra nodes that have a specific label key (whatever the values are). 
```

Comment:
If we put a note we should add wrap it with the {{% notice note %}}

Reply:
done